TombEngine/Game/collision/collide_room.cpp

```diff
@@ -738,6 +740,12 @@ int GetQuadrant(short angle)
 	return (unsigned short(angle + ANGLE(45.0f)) / ANGLE(90.0f));
 }
 
+bool TestNeighborRooms(int roomNumber0, int roomNumber1)
```

Comment:
Maybe it's better to name firstRoom and secondRoom instead of using numbers

Reply:
This is also by convention. If we want to change this we can, but I'd have to do it everywhere. :)